### ISSUE TITLE
About Target During Training

### ISSUE THREAD

**Issue by jaqub-manuel** [ultralytics/yolov5#2221]
Hey @glenn-jocher,

I think the target in training tells  how much of the data in the test will be tested in that epoch. I have 250 test data and the target is usually between 90-130 with 64 batches. So the first is best.pt with the next last.pt or a few epoks after the best. pt file is sometimes better than the last best.pt. I was wondering, how can I test all the data during training in one go, (mean targets=250) once trained, so I can increase the target to 250.
Thanks in advance ...

![2](https://user-images.githubusercontent.com/64843726/107936339-9542f100-6f93-11eb-875b-fd4ba2867371.PNG)
![1](https://user-images.githubusercontent.com/64843726/107936342-95db8780-6f93-11eb-9f0d-fc49dd88f7cb.PNG)

**Comment by glenn-jocher** [comment]
@jaqub-manuel targets column displays instances per batch.

test.py displayes total images and total instances seperately on same line as metrics.

**Comment by Sj1218305** [comment]
![image](https://user-images.githubusercontent.com/46301590/154247409-6a47198a-1dca-4927-87f1-b6499c1e0666.jpeg)
I have total 300 annotated images. But my targets are 301. How?
AFAIK Targets is the total test data that our model has. 
Also, I want to know how data is split into test and train dataset in yolov5

**Comment by glenn-jocher** [comment]
@Sj1218305 your YOLOv5 code is out of date, targets was renamed to labels a long time ago to avoid confusion. Your data.yaml contains split information.

To update YOLOv5:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!
